createmobilespec/createmobilespec.js

```diff
@@ -489,31 +495,32 @@ if (argv.plugman) {
 } else {
     // Create the project using "cordova create"
     myDelete(cli_project_dir);
-    console.log("Creating project mobilespec...");
-    shelljs.exec(cli + " create " + projectDirName + " org.apache.cordova.mobilespec MobileSpec_Tests --template cordova-mobile-spec" + path.sep + "www");
+    console.log("### Creating project mobilespec...");
+    executeShellCommand(cli + " create " + projectDirName + " org.apache.cordova.mobilespec MobileSpec_Tests --template cordova-mobile-spec" + path.sep + "www");
     shelljs.cp("-f", path.join(mobile_spec_git_dir, 'config.xml'), path.join(projectDirName, 'config.xml'));
 
     // Config.json file ---> linked to local libraries
     pushd(cli_project_dir);
 
     // Executing platform Add
-    console.log("Adding platforms...");
+    console.log("### Adding platforms...");
     [].concat(platforms).forEach(function (platform) {
-        console.log("Adding Platform: " + platform);
+        console.log("### Adding Platform: " + platform);
         var platformArg;
         if (argv.global) {
             platformArg = platform;
         } else {
             platformArg = join_paths([top_dir].concat(platform_layout[platform].bin));
             if (!fs.existsSync(platformArg)) {
-                couldNotFind(platformArg, platform);
+                couldNotFind(platform, platform);
                 platforms = platforms.filter(function (p) { return p != platform; });
                 return;
             }
         }
         console.log("platformArg: " + cli + " " + platformArg);
         var linkPlatformsFlag = (argv.link || argv.linkplatforms) ? ' --link' : '';
-        shelljs.exec(cli + ' platform add "' + platformArg + '" --verbose' + linkPlatformsFlag);
+        executeShellCommand(cli + ' platform add "' + platformArg + '" --verbose' + linkPlatformsFlag);
+        // TODO
```

Comment:
This TODO item resolved in PR #152 (which is currently awaiting review)

Reply:
```suggestion
```